docs/data/material/components/cards/cards.md

```diff
@@ -20,7 +20,7 @@ The Material UI Card component includes several complementary utility component
 - Card: a surface-level container for grouping related components.
 - Card Content: the wrapper for the Card content.
 - Card Header: an optional wrapper for the Card header.
-- Card Media: an optional container for displaying background images and gradient layers behind the Card Content.
+- Card Media:  displays images, videos, or iframes in a card with customizable size and centered styling.
```

Comment:
Description is correct. 👍 
Another suggestion is:
```
- Card Media: an optional versatile media wrapper that supports various media elements and background images with customizable styles and properties.
```

Reply:
ok would change the description , thanks for the suggestion and help